### PR TITLE
style: Demo Component overflow fix

### PR DESCRIPTION
**Pull request by MARKX97**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [X] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Demo Component overflow fix    |
| 🇨🇳 Chinese | Demo组件: 页面过窄超出样式修复    |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

**Review by MadCcc — APPROVED** [review]

File: .dumi/theme/common/styles/Demo.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ const GlobalDemoStyles: React.FC = () => {
           }
 
           .code-box-demo {
+            overflow: scroll;
```
Comment (afc163) [inline .dumi/theme/common/styles/Demo.tsx:41]: 不应该合这个，至少应该是 auto。

Comment (MARKX97) [reply .dumi/theme/common/styles/Demo.tsx:41]: 大佬，请问一下为什么不应该合呢，样式超出了诶

Comment (MARKX97) [reply .dumi/theme/common/styles/Demo.tsx:41]: 我再提一个auto吧

Comment (afc163) [reply .dumi/theme/common/styles/Demo.tsx:41]: https://github.com/ant-design/ant-design/pull/39898

会导致这个演示失效：https://preview-41209-ant-design.surge.sh/components/tabs#components-tabs-demo-custom-tab-bar

Comment (zombieJ) [reply .dumi/theme/common/styles/Demo.tsx:41]: #41225 先合了？现在每个 Demo 都多了一个框：
截屏2023-03-16 10 36 54

**Pull request merged**